_inc/lib/debugger/class-jetpack-cxn-tests.php

```diff
@@ -91,11 +91,30 @@ public static function increase_timeout() {
 	protected function test__check_if_connected() {
 		$name = __FUNCTION__;
 		if ( $this->helper_is_jetpack_connected() ) {
-			$result = self::passing_test( $name );
+			$message = sprintf(
+				'<p>%s</p><p>%s</p>',
```

Comment:
May need to check this to make sure the output works fine with CLI.

Reply:
What we could do is, when outputting the error message for the Site Health section, run the message through wpautop() to add paragraph tags to the text. The raw error message (and what the API and CLI output) is plain text, but add the HTML for Site Health?